CORE-DEVELOPMENT-README.md

```diff
@@ -0,0 +1,18 @@
+# CORE DEVELOPMENT GUIDE
+
+1. Clone this repository by using git clone https://github.com/bhanu951/drupal-core-development-project.git
```

Comment:
Are steps 1-4 needed for DDEV? Can you not use `composer create-project`? That command will do all that.

Reply:
once you clone this repo and execute `ddev composer install` drupal core will be downloaded into `repos/drupal` directory but it is having performance affect and taking atleast 2x time to setup.

Reply:
~once you clone this repo and execute ddev composer install drupal core will be downloaded into repos/drupal directory but it is having performance affect and taking atleast 2x time to setup.~

The above comment is invalid. We would be needing ` cd repos` and `git clone --branch=10.1.x https://git.drupalcode.org/project/drupal.git drupal`  mandatory as well. If not below error is thrown

> The `url `supplied for the path (repos/drupal) repository does not exist